src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
```

Comment:
Use a descriptive value for properties, one suggestion would be to use a const to store values that are repeated. This makes it easier to understand and maintain. For example:
const MALE = "m"
const FEMALE = "f"

people.filter(p => p.sex === FEMALE